### PR TITLE
UXD-188 ProgressBar

### PR DESCRIPTION
**Pull request by kcumlat**
### Purpose 🚀
To create ProgressBar component.
`<ProgressBar/>` shows the current progress of an action by relaying how much has already been done and how much is left
e.g. actions that could take some time (~15 secs)

### Notes ✏️
_details of code change / secondary purposes of this PR_

### Updates 📦
- [ ] MAJOR (breaking) change to _these packages_
- [ ] MINOR (backward compatible) change to _these packages_
- [ ] PATCH (bug fix) change to _these packages_

### Storybook 📕
http://storybooks.highbond-s3.com/paprika/UXD-188-progressbar

### Screenshots 📸
ProgressBar

### References 🔗
_relevant Jira ticket / GitHub issues_

### CODE REVIEW

**Comment by sunminsarahlee** [comment]
@karencumlat Looks great! I can't see live demo but can we have one gradient in the bar rather than two?  
Screen Shot 2020-09-02 at 4 12 13 PM

Expected design
Screen Shot 2020-09-02 at 4 12 54 PM

**Comment by oscarkwan** [comment]
1. The bar is off by a couple of pixels, you can see some white space on the left and the bottom line isn't aligned

Screen Shot 2020-09-03 at 1 04 20 PM

2. Can you investigate into using `role="progressbar"`, I'll leave it up to you to investigate.

**Comment by oscarkwan** [comment]
You can check out https://react-bootstrap.github.io/components/progress/ to see their html structure

File: packages/ProgressBar/src/ProgressBar.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  completed: 0,
+};
+
+function ProgressBar(props) {
+  const { a11yText, header, bodyText, completed, ...moreProps } = props;
+
+  const bestAria = a11yText || "Loading";
+
+  return (
+    <sc.ProgressBar data-pka-anchor="progress-bar" {...moreProps}>
```
Comment (oscarkwan) [inline packages/ProgressBar/src/ProgressBar.js:31]: I would expect the `data-pka-anchor` to be on the actual progress bar and not the container. What do you think?

Comment (kcumlat) [reply packages/ProgressBar/src/ProgressBar.js:31]: Yes, good point. I'll update this one 👍

**Comment by nahumzs** [comment]
If this implementation is ready, we should merge it and release so other developers can start integrating it, any extra work as screen reader implementation or specific a11y process can be added in parallel and in a new PR cc @oscarkwan.

Let me know @karencumlat  if this is ready.

**Comment by kcumlat** [comment]
> If this implementation is ready, we should merge it and release so other developers can start integrating it, any extra work as screen reader implementation or specific a11y process can be added in parallel and in a new PR cc @oscarkwan.
> 
> Let me know @karencumlat if this is ready.

Yes, it's ready. We can work on polish and a11y later 👍

**Review by nahumzs — APPROVED** [review]

**Review by nahumzs — APPROVED** [review]

**Pull request merged**